lib/porro/model.rb

```diff
@@ -39,7 +39,7 @@ def self.included(base)
     end
 
     def initialize(params = {})
-      self.attributes = params
+      self.attributes = attributes.merge(params.to_h)
```

Comment:
`nil` is not a problem for vanilla ruby `nil.to_h => {}`, however isn't it an user error if something non-hashy is passed to the model constructor?